pytorch_lightning/trainer/trainer.py

```diff
@@ -1208,7 +1214,7 @@ def _call_teardown_hook(self, model: "pl.LightningModule") -> None:
 
         if self.datamodule is not None:
             self.datamodule.teardown(stage=fn)
-        self.profiler.teardown(stage=fn)
```

Comment:
where is the profiler teardown happening now?

Reply:
It appears that `self.profiler.describe()` already includes a teardown call ([see this line](https://github.com/PyTorchLightning/pytorch-lightning/blob/d83dd7969db1857daac28ec352c177f41d4583eb/pytorch_lightning/profiler/base.py#L153)), so I think it is not necessary to invoke another one.

Reply:
ah right. that's something we'll clear up separately outside of this PR